### PR TITLE
Ensure Mapit and script file absent.

### PR DESCRIPTION
**Pull request by KludgeKML**
https://trello.com/c/LMRiT7yz/1581-retire-mapit-and-mapit-scripts

### CODE REVIEW

File: modules/govuk_ci/manifests/agent/postgresql.pp [hunk 1]
```diff
@@ -25,23 +22,15 @@
...
-  validate_slength($mapit_role_password, 20, 3)
   validate_slength($email_alert_api_role_password, 20, 3)
 
   postgresql::server::role { 'jenkins':
     password_hash => postgresql_password('jenkins', 'jenkins'),
     createdb      => true,
   }
 
-  # The mapit role needs to be a superuser in order to load the PostGIS extension for the test database.
-  postgresql::server::role { 'mapit':
```
[the remark this answers is not in the document]
Comment (KludgeKML) [reply modules/govuk_ci/manifests/agent/postgresql.pp:40]: Added back in with ensure absent.

[the remark this answers is not in the document]
Comment (KludgeKML) [reply modules/govuk_ci/manifests/agent/postgresql.pp]: Sadly, looks like our version of puppet doesn't support ensure for postgres roles, removed again.

[the remark this answers is not in the document]
Comment (1pretz1) [reply modules/govuk_ci/manifests/agent/postgresql.pp]: Thanks for attempting to remove! Having a lingering role isn't the end of the world, could remove it manually if we really care

[the remark this answers is not in the document]
Comment (KludgeKML) [reply modules/govuk_ci/manifests/agent/postgresql.pp]: Yeah, I was thinking "oh well, just devolves to click-ops (or cli-ops maybe)" At least we have a record of it.

**Review by 1pretz1 — APPROVED** [review]

**Pull request merged**